train.py

```diff
@@ -242,15 +251,16 @@ def main(args):
 		datasets.Labels(lang, bpe = bpe, name = f'bpe{i}', normalize_text_config = normalize_text_config) for i,
 		bpe in enumerate(args.bpe)
 	]
-	frontend = models.LogFilterBankFrontend(
+	frontend = getattr(models, args.frontend)(
 		args.num_input_features,
 		args.sample_rate,
 		args.window_size,
 		args.window_stride,
 		args.window,
 		dither = args.dither,
 		dither0 = args.dither0,
-		stft_mode = 'conv' if args.onnx else None
+		stft_mode = 'conv' if args.onnx else None,
+		fairseq_args = frontend_fariseq_args
```

Comment:
You can check if simply passing `**frontend_extra_args` would work (this would depend on whether fairseq conflicts with other kwargs, e.g. window/dither/dither0).
Alternatively you can put all these args into `frontend_extra_args = dict(dither = ..., dither0 = ...)` if LogFilterBank is used and load the args from the frontend checkpoint if not. That should work